grpc_message_getter.js

```diff
@@ -0,0 +1,292 @@
+    this.certPem = this._getRequiredProperty(properties,
+        'scalar.dl.client.cert_pem');
+    /** @const */
+    this.certHolderId = this._getRequiredProperty(properties,
+        'scalar.dl.client.cert_holder_id');
+    /** @const */
+    this.certVersion = properties['scalar.dl.client.cert_version'];
+
+    /** @const */
+    if (this._isNodeJsRuntime()) {
+      this.signer = new EllipticSigner(this.privateKeyPem);
+    } else {
+      this.signer = new WebCryptoSigner(this.privateKeyPem);
+    }
+  }
+
+  /**
+   * @param {Object} properties JSON Object used for setting client properties
+   * @param {string} name the name of the property to get
+   * @return {Object} The client property specified in the @name parameter
+   */
+  _getRequiredProperty(properties, name) {
+    const value = properties[name];
+    if (!value) {
+      throw new ClientError(
+          StatusCode.CLIENT_IO_ERROR,
+          `property '${name}' is required`,
+      );
+    }
+    return value;
+  }
+
+  /**
+   * @return {Uint8Array}
+   */
+  async getCertificateRegistrationRequest() {
+    const contractCertHolderIdEncoded = new TextEncoder('utf-8').encode(
+        this.certHolderId);
+    const contractArgumentEncoded = new TextEncoder('utf-8').encode(
+        this.certVersion);
+    const contractCertPemEncoded = new TextEncoder('utf-8').encode(
+        this.certPem);
+
+    const buffer = new Uint8Array(
+        contractCertHolderIdEncoded.byteLength
+        + contractArgumentEncoded.byteLength
+        + contractCertPemEncoded.byteLength);
+
+    let offset = 0;
+    buffer.set(contractCertHolderIdEncoded, offset);
+    offset += contractCertHolderIdEncoded.byteLength;
+    buffer.set(contractArgumentEncoded, offset);
+    offset += contractArgumentEncoded.byteLength;
+    buffer.set(contractCertPemEncoded, offset);
+
+    return buffer
+  }
+
+  /**
+   * @param {number} functionId of the function
+   * @param {string} functionBinaryName of the function
+   * @param {Uint8Array} functionBytes of the function
+   * @return {Uint8Array}
+   */
+  async getFunctionRegistrationRequest(functionId, functionBinaryName,
+      functionBytes) {
+    if (!(functionBytes instanceof Uint8Array)) {
+      throw new ClientError(
+          StatusCode.CLIENT_IO_ERROR,
+          'parameter functionBytes is not a \'Uint8Array\'',
+      );
+    }
+    const functionIdEncoded = new TextEncoder('utf-8').encode(
+        functionId.toString());
+    const functionBinaryNameEncoded = new TextEncoder('utf-8').encode(
+        functionBinaryName);
+
+    const buffer = new Uint8Array(
+        functionIdEncoded.byteLength + functionBinaryNameEncoded.byteLength
+        + functionBytes.byteLength);
+
+    let offset = 0;
+    buffer.set(functionIdEncoded, offset);
+    offset += functionIdEncoded.byteLength;
+    buffer.set(functionBinaryNameEncoded, offset);
+    offset += functionBinaryNameEncoded.byteLength;
+    buffer.set(functionBytes, offset);
```

Comment:
Since, functionBytes is in `Uint8Array`, therefore it is not encoded... same to how we did for [builder](https://github.com/scalar-labs/scalardl-javascript-sdk-base/blob/master/request/builder.js#L234).